code/drasil-example/Drasil/SSP/Unitals.hs

```diff
@@ -401,12 +401,14 @@ totStress = uc' "sigma" (cn' "total stress")
   "the total force per area acting on the soil mass" lSigma pascal
 
 effectiveStress = uc' "sigma'" (cn' "effective stress")
-  ("the stress in a soil mass that is effective in causing volume changes; " ++
-   "represents the average stress carried by the soil skeleton")
+  ("the stress in a soil mass that is effective in causing volume changes " ++
+   "and mobilizes the shear strength arising from friction; represents the " ++
+   "average stress carried by the soil skeleton")
   (prime lSigma) pascal
 
 effNormStress = uc' "sigmaN'" (cn' "effective normal stress")
-  ("the normal stress in a soil mass that is effective in causing volume changes; " ++
+  ("the normal stress in a soil mass that is effective in causing volume " ++
+   "changes and mobilizes the shear strength arising from friction; " ++ 
```

Comment:
@samm82, I was thinking  you would leave this one alone, because it was already correct.  The normal stress is just the normal stress, so we shouldn't mention the shear strength.  Normal stresses cause volume changes.  Shear stresses cause dilations (changes in angles).  Please modify this one back to the way you had it originally.  😄

Reply:
Yikes 😬 my bad - I'll get on that! 😂